jvm/src/main/scala/mouse/jvm.scala

```diff
@@ -0,0 +1,4 @@
+package mouse
+
+trait AllJvmSyntax
+  extends StringJvmSyntax
```

Comment:
The `all` import needs to expose all of the syntax traits, both the shared and JVM/JS specific ones. My solution here is for both the `js` and `jvm` configurations to have a dedicated `AllSyntax` trait. This is then mixed in to the `all` import.

For the JVM, this becomes:

```scala
   object all extends AllSharedSyntax with AllJvmSyntax
```

For JS, it is:

```scala
   object all extends AllSharedSyntax with AllJsSyntax 
```

Since there are no JS-specific syntax classes, the `AllJsSyntax` trait currently does nothing. I've added it for the purposes of symmetry. Happy to remove it if all it does is add confusion.